site/src/docs/components/login/usage.mdx

```diff
@@ -556,6 +620,41 @@ All `Session poller` settings, properties, methods and signals are detailed on t
 
 Detailed information about these hooks is listed on the <ApiPageAnchorLink anchor="session-poller-hooks">API page</ApiPageAnchorLink>.
 
+#### GraphQL module
+
+Loading data from a graphQL server may require user authentication and api tokens. The GraphQL module can be set to listen to the Api token client and start querying when api tokens are ready. It can pick the correct api token and set it to the query headers.
```

Comment:
one inconsistent Api, while others are written api

Reply:
This "Api" is intentionally capitalized because it's part of  the module name "Api token client"